beep/tests/test_end_to_end.py

```diff
@@ -28,7 +27,7 @@ def setUp(self):
             kinesis = boto3.client('kinesis')
             response = kinesis.list_streams()
             self.events_mode = 'test'
-        except NoRegionError or NoCredentialsError as e:
+        except Exception as e:
```

Comment:
Can we raise a warning here?  Since the only conceivable errors in the try block are those related to AWS, I'm fine to use a blank exception, but if someone is trying to use the AWS functionality and hasn't configured it correctly, we'd want them to know, rather than erroring silently.

Reply:
Good idea, I am including a warning in there